xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -413,7 +422,12 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
 ##
 ##
 #if(!$replied)
-  #displayCommentForm()
+  #if($xwiki.hasAccessLevel('comment'))
+    #displayCommentButton()
+    <div id="commentFormPlaceholder" class="comment-form-hidden">
```

Comment:
Note that you don't need a specific class for this we already have a common class `hidden` used for that purpose.

Reply:
Indeed, thanks even better, thanks. Class `comment-form-hidden` replaced by `hidden`.